papiea-sdk/typescript/__tests__/intentful_workflow_tests/intentful_workflow.test.ts

```
@@ -71,7 +71,10 @@ describe("Intentful Workflow tests", () => {
         }
         first_provider_to_delete_entites = []
         second_provider_to_delete_entites = []
-        await providerApiAdmin.delete(`${first_provider_prefix}/${provider_version}`)
+        try {
+            await providerApiAdmin.delete(`${first_provider_prefix}/${provider_version}`)
+        } catch (e) {
+        }
```

Just silently ignoring may miss problems, no?

---

This a cleanup code, pretty sure the deletion problems should be signified by earlier tests (entity.api.test.ts)

---

In case there's some problem with the cleanup, other tests may fail without an apparent reason. I think at least logging would be helpful.